lib/upperkut/processor.rb

```diff
@@ -11,10 +11,9 @@ def initialize(worker, logger = Logging.logger)
 
     def process
       items = @worker.fetch_items.freeze
-      items_body = items.map(&:body)
 
       @worker.server_middlewares.invoke(@worker, items) do
-        @worker_instance.perform(items_body.dup)
+        @worker_instance.perform(items)
```

Comment:
`items.dup`

cause, the worker should mutate items object.

Reply:
I was expecting `Item`s to be immutable, what is not true due to an implementation detail on `ScheduledStrategy` that I'll soon review.